.github/workflows/lint.yml

```
@@ -26,4 +26,23 @@
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('apps/backend/pom.xml') }}
       - name: Run Checkstyle
-        run: cd apps/backend && mvn checkstyle:check
+        run: cd apps/backend && mvn checkstyle:check
+  frontend-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)